apps/prairielearn/src/lib/editors.test.ts

```diff
@@ -178,4 +178,42 @@ describe('editors', () => {
       });
     });
   });
+  describe('getPropertiesForAdd', () => {
```

Comment:
👀 I'd expect this to have the name of the function.

```suggestion

  describe('propertyValueWithDefault', () => {
```